### PR TITLE
Dragon ascii art #3

### PR DESCRIPTION
**Pull request by nxb9626**
#3 

Implemented dragon ascii

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /target
 .DS_Store
 info.json
+.vscode
```
Comment (peterkos) [inline .gitignore]: CleanShot 2021-12-27 at 01 23 18

File: src/dragon.rs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                            `/ho`                 
+               `:-..```   `-oy+`                  
+             `/syhyyso++/+so/``                   
+            .syyys---::/--`                       
+           `/-.                                                                     
+           _         _           _ 
+  ___ __ _| | ___ __| |_ __   __| |
+ / __/ _` | |/ __/ _` | '_ \\ / _` |
+| (_| (_| | | (_| (_| | | | | (_| |
+ \\___\\__,_|_|\\___\\__,_|_| |_|\\__,_|
```
Comment (peterkos) [inline src/dragon.rs:33]: maybe one thats a bit more readable?

File: src/dragon.rs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+           "   
+        )
+    }
```
Comment (peterkos) [inline src/dragon.rs]: memed-io-output

File: src/dragon.rs [hunk 4]
```diff
@@ -0,0 +1,37 @@
+/// Original dragon image modified from:
+/// https://www.rawpixel.com/image/581164/hand-drawn-dragon
+/// using the following webiste:
+/// https://www.text-image.com/convert/ascii.html
+pub fn loading(){
```
Comment (peterkos) [inline src/dragon.rs]: nit: space before brace
```suggestion
pub fn loading() {
```

File: src/dragon.rs (hunk 2 above)
Comment (nxb9626) [reply src/dragon.rs:33]: escape '\'  are what's causing this to look odd, without them it won't compile. It prints out as below. 
image

**Review by peterkos — APPROVED** [review]

File: src/dragon.rs (hunk 2 above)
Comment (peterkos) [reply src/dragon.rs:33]: Ahhhhhh!

**Pull request merged**